src/digital.c

```diff
@@ -1,52 +1,140 @@
 #include "digital.h"
+#include <stdbool.h>
 
-void digital_write(uint8_t pin, uint8_t value)
+static void pinWrite(uint8_t pin, uint8_t value, bool dir)
```

Comment:
parameter name ?
The pinWrite sounds weird with the direction param
maybe call is 'set_dir' or something less ambiguous ?

Reply:
Sure